### ISSUE TITLE
ProjectTemplate example fails in episode 16

### ISSUE THREAD

**Issue by jcoliver** [swcarpentry/r-novice-gapminder#501]
At the beginning of episode 16 (Writing Good Software):

```
create.project("../my_project", merge.strategy = "allow.non.conflict")
Error in .create.project.existing(project.name, merge.strategy, template,  : 
  Creating a project in  ../my_project  would overwrite the following existing files/directories:
 data
```

Likely due to creation of `data` directory.

**Comment by jcoliver** [comment]
Could be solved by using a different name for the new project, i.e. `../my_project_2`

**Comment by jcoliver** [comment]
Addressed by #506
